experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -47,13 +48,34 @@ import {
 import { makeTree } from "./editableTree";
 import { ProxyTarget } from "./ProxyTarget";
 
+/* eslint-disable no-param-reassign */
+/* eslint-disable no-bitwise */
+
+/**
+ * Coerces the given 'index' to an Int32.  This includes the following behaviors:
```

Comment:
This can produce non-integer values, infinities, and values larger than 32 bits. Also one of the main points of this is to convert negative indexes into positive indexes relative to the end, so "int32" being a singed type seems extra misleading.

I think talking about the semantics would be better than refering to a not very relevent type.

Reply:
Agree.